cmd/kube-proxy/app/server.go

```diff
@@ -127,16 +130,24 @@ func (s *ProxyServer) Run(_ []string) error {
 		glog.V(2).Infof("Running in resource-only container %q", s.ResourceContainer)
 	}
 
+	if s.Master != "" {
+		if len(s.APIServerList) != 0 {
+			glog.Warningf("Both --mater and --api-servers set. Using --api-servers")
```

Comment:
Can we hard fail if they don't match?  Return a usage error?

On Fri, Sep 11, 2015 at 12:01 PM, Eric Paris <notifications@github.com>
wrote:

> In cmd/kube-proxy/app/server.go
> <https://github.com/kubernetes/kubernetes/pull/13555#discussion_r39286559>
> :
>
> > @@ -127,16 +130,24 @@ func (s *ProxyServer) Run(_ []string) error {
> >  		glog.V(2).Infof("Running in resource-only container %q", s.ResourceContainer)
> >  	}
> >
> > +	if s.Master != "" {
> > +		if len(s.APIServerList) != 0 {
> > +			glog.Warningf("Both --mater and --api-servers set. Using --api-servers")
>
> (not that the user in that case will get both this error message and the
> error message because they used a deprecate flag)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13555/files#r39286559>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift